src/contracts/Tab/ConditionalTabInterface.php

```diff
@@ -10,16 +10,14 @@
 
 /**
  * Conditional Tab interface needs to be implemented by tabs,
- * which needs to be evaluate depends on context.
+ * which needs to be evaluated depends on context.
```

Comment:
We can go an extra mile if we are improving wording here... ;)
```suggestion
 * which needs to be evaluated depending on context.
```